### ISSUE TITLE
gTile crashes when disconnecting one of the monitors

### ISSUE THREAD

**Issue by crabtree** [gTile/gTile#165]
When working on multiple monitors setup and disconnecting/connecting one of the monitors gTile plugin crashes. I've debugged the problem and the logs for the gnome-shell says:
```
JS ERROR: TypeError: grid is undefined
showTiling@/home/crabtree/.local/share/gnome-shell/extensions/gTile@vibou/extension.js:897:29
toggleTiling@/home/crabtree/.local/share/gnome-shell/extensions/gTile@vibou/extension.js:942:9
 _onButtonPress@/home/crabtree/.local/share/gnome-shell/extensions/gTile@vibou/extension.js:541:9
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
```

It seems that the grids collection is not refreshed after the state of the monitors is changed. I've managed to solve the problem by connecting to the event `'monitors-changed'` and re-initializing the girds when it happens.

**Comment by r3dpointsp** [comment]
I have a laptop + 21:9 external monitor. I normally run with laptop lid closed and just use the 21:9 monitor. gTile worked fine on Ubuntu 20.04.1/Gnome Shell 3.36. Today, after clean install of Ubuntu 20.10/Gnome Shell 3.38, when I close the laptop lid, gTile stops working. gTile still in activity bar, and gTile settings are accessible in Extensions. However, super+enter does nothing and clicking on gTile icon does nothing. When I have laptop lid open + 21:9 external monitor, gTile appears to work just fine.

**Comment by scherepanov** [comment]
@r3dpointsp can you try to run gTile with PR #166 and see if it fixes your problem

**Comment by crabtree** [comment]
@r3dpointsp yes, that is the problem I've got and I try to fix in PR #166.

**Comment by r3dpointsp** [comment]
> @r3dpointsp can you try to run gTile with PR #166 and see if it fixes your problem

Works now with PR #166. It appears flaky; however, at least now I can 

```bash
gnome-extensions disable gTile@vibou
gnome-extensions enable gTile@vibou
```

and get gTile to a working state if super+enter doesn't bring up gTile. After the commands above super+enter brings up gTile.
